portal/_header.php

```diff
@@ -139,9 +139,9 @@ class="fa fa-book"></i> <span><?php echo xlt('Accountings'); ?></span>
                             <li data-toggle="pill"><a href="#ledgerpanel" data-toggle="collapse"
                                 data-parent="#panelgroup"> <i class="fa fa-folder-open"></i> <span><?php echo xlt('Ledger'); ?></span>
                             </a></li>
-                            <!-- <li data-toggle="pill"><a href="#paymentpanel" data-toggle="collapse"
-                                data-parent="#panelgroup"> <i class="fa fa-credit-card"></i> <span><?php //echo xlt('Make Payment'); ?></span>
-                            </a></li> -->
+                            <li data-toggle="pill"><a href="#paymentpanel" data-toggle="collapse"
+                                data-parent="#panelgroup"> <i class="fa fa-credit-card"></i> <span><?php echo xlt('Make Payment'); ?></span>
+                            </a></li><!--  -->
```

Comment:
Seems odd that this comment code is here since now just commented out 1 empty space.

Reply:
Too lazy to delete comment so I just pulled out of way. I do that for stuff I comment in and out often.

Reply:
Would it make sense to make this a option (ie. global selection)?

Reply:
Will do then I would feel better about turning it on.